### PR TITLE
@craigspaeth - Refactor and js implementation

### PR DESCRIPTION
**Pull request by xtina-starr**
Hey Craig, sorry for the loaded PR. I probably should have committed and pushed a little bit more frequently. This includes the feedback you provided from the last PR along with new JS implementation for the slideshow and click event to expand the content that I mentioned earlier.

chinese splash page

### CODE REVIEW

File: components/layout/index.coffee [hunk 1]
```diff
@@ -1,4 +1,19 @@
 $ = require 'jquery'
 
-# $ ->
-#   alert 'hi'
+$ ->
+
+  # target clicked, callback -> close all sections then slideDown clicked element
+  $('.js-display-content').click (e) ->
```
Comment (broskoski) [inline components/layout/index.coffee:6]: Much DRYer!

File: components/body/index.styl [hunk 2]
```diff
@@ -65,22 +87,36 @@
...
-.section-three-content p, .section-four-content p
-  margin-bottom 60px
-  line-height 1.5
-
-// .section-three-content
-  // display none
-
-.launch-app-store
-  width 205px
+.avant-garde-button-black
```
Comment (craigspaeth) [inline components/body/index.styl:93]: Styles like this and `.carousel-track__image` are legit, but they belong outside of the body component b/c they're used outside of the scope of just the body (or if they aren't then they should be namespaced by `body`). You could move these to stylesheets inside the "global" layout component e.g. components/layout/buttons.styl components/layout/carousel.styl (what we do in Force/MG) or extract them as their own components e.g. components/buttons/index.styl, components/carousel/index.styl

Not a big deal, but just hoping erring on overly scrutinizing is helpful in understanding how we organize components.

File: components/layout/index.coffee [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+
+  slideSwitch = ->
+    $active = $('figure.carousel-track__image  .slideshow-is-active')
+
+    $next = if $active.next().length then $active.next() else $('figure.carousel-track__image img:first')
+
+    $next.addClass('slideshow-is-active')
+    $active.removeClass('slideshow-is-active')
+
+  setInterval( slideSwitch, 2000)
```
Comment (craigspaeth) [inline components/layout/index.coffee:19]: Nice!

**Comment by craigspaeth** [comment]
Nice! Making some good progress!

As the body component gets bigger you may want to consider how you would break that up into smaller pieces. e.g. a generic "section" component that involves jade mixins and CSS classes that are useful in DRYing up common patterns across sections and maybe a "section1", "section2" etc. component for each implementation. Again, components are just a way to organize related UI code (stylesheets, templates, view code, etc.) so there's no hard fast rules, but it's good to break down large pieces in this fashion for ease of reuse and to ensure you avoid CSS spaghetti by name-spacing.

**Pull request merged**